src/program/lwaftr/bench/bench.lua

```diff
@@ -31,7 +31,9 @@ function parse_args(args)
    function handlers.b(arg) opts.bench_file = arg end
    function handlers.y() opts.hydra = true end
    function handlers.h() show_usage(0) end
-   function handlers.reconfigurable() opts.reconfigurable = true end
+   function handlers.reconfigurable()
+      io.stderr:write("Warning: --reconfigurable flag has been deprecated.\n")
```

Comment:
Let's remove this one entirely, as this is an internal script that's not user-facing.